x/auth/ante/sigverify.go

```diff
@@ -258,6 +293,11 @@ func (svd SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simul
 
 	utx, ok := tx.(sdk.TxWithUnordered)
 	isUnordered := ok && utx.GetUnordered()
+	unorderedEnabled := svd.ak.IsUnorderedTransactionsEnabled()
```

Comment:
should we just use ak and not have the `unorderedTxManager`field at all?

Reply:
yeah i can do that instead